extension/framework/framework.go

```diff
@@ -33,9 +35,28 @@ var log = logging.MustGetLogger("extest")
 func TestExtensions(c *cli.Context) {
 	buflog.SetUpDefaultLogging()
 
+	var config *util.Config
+	configFilePath := c.String("config-file")
+
+	if configFilePath != "" && !c.Bool("verbose") {
+		config = util.GetConfig()
+		err := config.ReadConfig(configFilePath)
+		if err != nil {
+			log.Error(fmt.Sprintf("Failed to read config from path %s", configFilePath))
+			os.Exit(1)
+		}
+
+		err = logg.SetUpLogging(config)
+		if err != nil {
+			log.Error(fmt.Sprintf("Failed to set up logging: %v", err))
+			os.Exit(1)
+		}
+	}
+
 	testFiles := getTestFiles(c.Args())
 
-	returnCode := RunTests(testFiles, c.Bool("verbose"))
+	//logging from config is a limited printAllLogs option
+	returnCode := RunTests(testFiles, c.Bool("verbose") || config != nil)
```

Comment:
It's because I didn't want to add unnecessary logic to this code. We don't want to log config logs via buffer log, because then we will need to rewrite gohan_log (or logg) functionality in it. For me there is no point in using both config and verbose option - you can set in config default logging level to DEBUG and have verbose output.